workflow-dispatch/action.yml

```
@@ -0,0 +1,25 @@
+name: Workflow Dispatch
+description: Triggers a workflow with the workflow_dispatch event.
+inputs:
+  owner:
+    description: "Name of the repository owner in which the workflow resides. Defaults to the one who owns the workflow in which this action is running."
+    required: false
+  repo:
+    description: "Name of the repository in which the workflow resides. Defaults to the one in which the workflow currently running resides."
+    required: false
```

Just an idea, but what would you think of merging those two?
```suggestion
  repo:
    description: "Full name of the repository in which the workflow resides in form of `${owner}/${projectName}`. Defaults to the one in which the workflow currently running resides."
    required: false
```

---

I think I prefer leaving them separated 🤷‍♂️

---

All right then